dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/DataIntegrityController.java

```diff
@@ -88,4 +95,53 @@ public WebMessage runDataIntegrity(
         return jobConfigurationReport( config );
     }
 
+    @GetMapping
+    @ResponseBody
+    public Collection<DataIntegrityCheck> getAvailableChecks()
+    {
+        return dataIntegrityService.getDataIntegrityChecks();
+    }
+
+    @GetMapping( "/summary" )
+    @ResponseBody
```

Comment:
do we need `@PreAuthorize( "hasRole('ALL') or hasRole('F_PERFORM_MAINTENANCE')" )` for those new endpoints ?

Reply:
Haha, yeah, wanted to add them as soon as I got the tests working and then forgot about it. Good catch!

Reply:
ok, other than that, codes look nice as always :)

Reply:
added the role auth